### PR TITLE
Feature/pending edits ux

### LINKED ISSUE #193: Show pending edits by fading the event body
like for pending messages

### PR DESCRIPTION
**Pull request by BillCarsonFr**
Show pending edits by fading the event body 
Fixes #193
 + Show replace events in timeline when show hidden event option is selected in lab

### CODE REVIEW

File: vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/MessageInformationData.kt [hunk 1]
```diff
@@ -31,7 +31,8 @@ data class MessageInformationData(
         val showInformation: Boolean = true,
         /*List of reactions (emoji,count,isSelected)*/
         var orderedReactionList: List<ReactionInfoData>? = null,
-        var hasBeenEdited: Boolean = false
+        var hasBeenEdited: Boolean = false,
```
Comment (ganfra) [inline vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/MessageInformationData.kt]: we should only get val on this class, could you please change that on the 3 fields?

File: vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/AbsMessageItem.kt [hunk 2]
```diff
@@ -163,7 +164,9 @@ abstract class AbsMessageItem<H : AbsMessageItem.Holder> : BaseEventItem<H>() {
 
     protected fun renderSendState(root: View, textView: TextView?) {
         root.isClickable = informationData.sendState.isSent()
-        textView?.setTextColor(colorProvider.getMessageTextColor(informationData.sendState))
+        val state = SendState.UNSENT.takeIf { informationData.hasPendingEdits }
```
Comment (ganfra) [inline vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/AbsMessageItem.kt]: I think a basic If else would be more readable

Comment (BillCarsonFr) [reply vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/AbsMessageItem.kt]: done

File: vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/MessageInformationData.kt (hunk 1 above)
Comment (BillCarsonFr) [reply vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/MessageInformationData.kt]: done

**Pull request merged**